cmd/create.go

```diff
@@ -277,6 +284,17 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 				"firewall-port": inletsPort,
 			},
 		}, nil
+	} else if provider == "ec2" {
+		return &provision.BasicHost{
+			Name:     name,
+			OS:       "ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-20191114",
```

Comment:
Is there a webpage we can link to in a comment, i.e. if we want to trace the image and update to a newer version in the future?

Reply:
Yes there is, I will add it